### DISCUSSION TITLE
oh-my-posh, Ubuntu 22.04 (bash) and virtualenv not working for me

### DISCUSSION THREAD

**Discussion by 0xab3d** [JanDeDobbeleer/oh-my-posh#2297] [Troubleshoot] [answered]
Hello,

Could someone assist me with how can I show **_(venv) directory_** when I enter the virtual environment?

**Comment by JanDeDobbeleer** [comment, marked as answer]
Does your config/theme contain a [python](https://ohmyposh.dev/docs/segments/python) segment? Might be useful to copy/paste the output of `oh-my-posh debug` in case it does.

**Reply by 0xab3d** [reply]
Now it is added :)

**Comment by GregWoods** [comment]
I arrived here with the same question, but this brief discussion was no help. I have no idea what was added, and the link is dead (because MS like to do that!) I#d suggest StackOverflow style replies in future. :)

**Reply by JanDeDobbeleer** [reply]
@GregWoods the docs are broken because I moved that page. It's [here](https://ohmyposh.dev/docs/segments/languages/python) now.

**Reply by GregWoods** [reply]
Superb. Thank you 😃
